Resources/Audio/Effects/Footsteps/attributions.yml

```diff
@@ -32,3 +32,11 @@
   license: "CC-BY-SA-3.0"
   copyright: "Taken and modified from tgstation (clownstep 1 and 2) by brainfood1183 (github)"
   source: "https://github.com/tgstation/tgstation/tree/f8ee37afc00bce1ad421615eaa0e4cbddd5eea90/sound/effects"
+
+- files:
+  - snake1.ogg
+  - snake2.ogg
+  - snake3.ogg
+  license: "CC-BY-SA-3.0"
```

Comment:
Where is this license listed?

Reply:
No licence is listed here, the author of the sounds wrote this:
Here you can download and listen to The Sound of Snakes online for free. Use them to edit and insert them into your video clips, as soundtrack, or for any other purpose. In addition, you can give a listen to your children.
*
Sounds of snakes in mp3 format download for free and without registration. You can use them for editing videos, creating music and songs, for ringtone, designing presentations, developing games or applications.
*
As I understand such a licence is suitable, or do I need to specify a special licence?

Reply:
Use custom and specify as such rather than putting an incorrect license on.